pkg/scaler/utils.go

```diff
@@ -57,3 +59,25 @@ func isRateLimited(err error) bool {
 	const prefix = "rate: "
 	return strings.HasPrefix(err.Error(), prefix)
 }
+
+func shootWokeUp(old, new *gardenerv1alpha1.Cluster) bool {
```

Comment:
Shouldn't this be more `shootInHibernationTransition`?

Reply:
This function is only to check the type of `update` event on the `cluster`. Returns true only if the `status` changed from "sleeping/waking up" to "awake/ready". I will change the name of the function to better reflect its purpose.